docs/Operators.md

```diff
@@ -7334,13 +7334,14 @@ expect(node, inputs=[x], outputs=[y], name="test_depthtospace_example")
 
 ### <a name="DequantizeLinear"></a><a name="dequantizelinear">**DequantizeLinear**</a>
 
-  The linear dequantization operator. It consumes a quantized tensor, a scale, and a zero point to compute the full precision tensor.
-  The dequantization formula is `y = (x - x_zero_point) * x_scale`. `x_scale` and `x_zero_point` must have same shape, and can be either a scalar
-  for per-tensor / per layer quantization, or a 1-D tensor for per-axis quantization.
-  `x_zero_point` and `x` must have same type. `x` and `y` must have same shape. In the case of dequantizing int32,
+  The linear dequantization operator. It consumes a quantized tensor, a scale, and a zero point to compute the full-precision tensor.
+  The dequantization formula is `y = (x - x_zero_point) * x_scale`. `x_scale` and `x_zero_point` must have the same shape, determining the quantization's granularity:
+  a scalar for per-tensor/per-layer quantization, a 1-D tensor for per-axis quantization, or have a rank identical to the input for blocked quantization.
+  Blocked quantization is not supported for 1-D input tensors. See QuantizeLinear for details on quantization granularity.
```

Comment:
Why is it excluded for 1-D input tensors?

Reply:
See discussion here: https://github.com/onnx/onnx/pull/5812/files#r1441443181